tests/manual-test-cases/Group17-ManualTTY/17-1-TTY-Tests.robot

```diff
@@ -32,4 +32,28 @@ Docker run -it command that doesn't stop
     ${out}=  Run  docker ${params} logs ${containerID}
     Should Contain  ${out}  Mem:
     Should Contain  ${out}  CPU:
-    Should Contain  ${out}  Load average:
\ No newline at end of file
+    Should Contain  ${out}  Load average:
+
+Docker run with -i
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} run -i busybox dmesg
```

Comment:
can we use the following or similar - relying on dmesg output stability make break us in the future:
`run -i busybox /bin/ash -c "dmesg;echo complete"`